src/style.css

```diff
@@ -1 +1,119 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+body {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
You've done a good job specifying a fallback font in case the main font doesn't load. This is a good practice as it ensures that some font will be displayed, improving your website's resilience and usability. Keep it up!